base/server/etc/default.cfg

```diff
@@ -384,6 +384,8 @@ pki_request_id_length=128
 pki_security_domain_setup=True
 pki_registry_enable=True
 
+pki_client_verify_cert=False
```

Comment:
I think we should disable revocation check only if it's causing a chicken and egg problem (e.g. installing CA with certs pointing to its built-in OCSP) so I'm not sure if we should apply the same default policy for all installation cases.

Reply:
I have update to true. With latest changes to default OCSP there are no problem in workflows.